senior-knights/course-schedulizer
# Add timestamp

This PR adds a field called LastEditTimestamp to the export so that users can sort the spreadsheet by the timestamp to see which sections were most recently edited.

To Test:
- Upload a schedule
- Update a few of the sections
- Export the full spreadsheet
- Open the spreadsheet in Excel and sort by the LastEditTimestamp field. Your edited sections should be at the top
- Reimport the spreadsheet and edit a few different sections
- Export and sort again. Notice that your most recent changes are on top, then the changes from the previous export, then the unchanged sections

Closes #119
